Source/WebCore/platform/ThreadSafeDataBuffer.h

```diff
@@ -56,6 +64,8 @@ friend class ThreadSafeDataBuffer;
 };
 
 class ThreadSafeDataBuffer {
+private:
```

Comment:
nit: not needed, since class has private by default.  Also, friend can go below with the other private stuff.